exec/java-exec/src/test/java/org/apache/drill/exec/vector/complex/writer/TestJsonNanInf.java

```diff
@@ -236,6 +236,22 @@ public void testConvertFromJsonFunction() throws Exception {
 
 
 
+  @Test
+  public void testLargeStringBinary() throws Exception {
+    String chunk = "0123456789";
+    String data = "";
+    for (int i = 0; i < 1000; i++) {
+      data += chunk;
```

Comment:
It is preferable to change such string concatenations with `StringBuilder#append(String)`. (`String data` --> `StringBuilder data`).